rDNA/rDNA/R/rDNA.R

```diff
@@ -4848,9 +4851,9 @@ print.dna_phaseTransitions <- function(x, ...) {
 #'   \code{"states"}.
 #'
 #' @author Philip Leifeld, Kristijan Garic
-#' @importFrom ggplot2 autoplot ggplot aes_string geom_line geom_point xlab ylab
+#' @importFrom ggplot2 autoplot ggplot aes geom_line geom_point xlab ylab
 #'   labs ggtitle theme_bw theme arrow unit scale_shape_manual element_text
-#'   scale_x_datetime scale_colour_manual guides
+#'   scale_x_datetime scale_colour_manual guides rlang
```

Comment:
Is `rlang` an object or function exported by `ggplot2`? I thought it was an independent package. You need to add something like this to every function that uses `.data`: `@importFrom rlang .data`. I.e., also to the other autoplot methods that use this. Then recompile the documentation.